Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM --platform=$BUILDPLATFORM node:19.7.0-alpine3.16 AS client-builder
 
+# Required for react-create-app to work properly.
+ENV PUBLIC_URL=.
+ENV BROWSER=none
```

Comment:
That is only needed when you work locally and run the dev mode of the UI. By default, a browser is open pointing to localhost:3000 which we found pretty annoying :)

Reply:
Gotcha. I'll send another PR to replace cra with vite tho.
For now this works fine.